_slides/05-dg630s.md

```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Dallas Guillory
```

Comment:
Maybe you can add something else here

```suggestion
Your name
```

Reply:
![Patch Panel](https://user-images.githubusercontent.com/35269554/228606877-3a043e27-fdda-4168-b16f-e865eedbe72a.jpg)